src/Pizzeria.Common/Pizzeria.Common.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net9.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
This PropertyGroup can be removed as it's already globally set in Directory.Build.props